docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
+  * Provide Issue and PR templates to improve the contributor experience.
+
+
+# Creating your community space
+
+Each Ansible ecosystem project should have ways to communicate and collaborate
+ with community members. The following sections cover ways to interact
+ and build your project community.
+
+## Matrix
+Our preferred chat option for Ansible projects is Matrix. Your matrix room name
+should match your project name. You can set up your own matrix room or 
+contact the community team in the forum to be officially added to the Ansible matrix space.
+By doing so your project gets:
+
+- Added to the Space (which functions as the directory)
+- An ansible.com room alias
+- The Admin bot to give you moderation/spam coverage
+- A Matrix meetbot to help you create meeting summaries and logs, automatically posted to the forum
+- (eventually) A room icon based on our template
+
+Contact the community team in the forum to coordinate a new Matrix room for your project.
+
+## Using the Ansible forum
+Ansible introduced the [Ansible Community Forum](https://forum.ansible.com/).
+This is a great place to interact with community members and grow your own project
+community. We have a [set of videos](https://forum.ansible.com/t/discourse-video-demos/102)
+ to help you learn how to use the forum, as well as 
+ [some other tips](https://forum.ansible.com/tag/forum-tips). 
+
+Reach out to the community team to request new tag(s) for your project in the forum.
+
+## Working Groups
+Not every project requires a working group. We advise projects to use
+the Ansible forum and GitHub issues as much as possible
+to be accessible to all timezones. 
+
+A working group is your group of contributors, along with ways to coordinate
+your work. This includes a tag or group for the forum and optionally a matrix room. 
+ 
+Synchronous project meetings are an option and you can use the 
+[Forum Events calendar](https://forum.ansible.com/c/events/8) to announce your meetings. 
+Be aware that any real-time meetings will tend to exclude contributors not in or around the 
+meeting timezone.
+
+If you do decide to have a synchronous meeting, add your working group meeting 
+time to the [shared Ansible ICS calender in GitHub](https://github.com/ansible/community/tree/main/meetings).
+
+You can also include your meeting calendar and agenda in the forum. 
+See [the DaWGs agenda](https://forum.ansible.com/t/documentation-working-group-agenda/153)
+for one possible way to do this.
+
+If you run meetings on Matrix, you can invite the Ansible Matrix bot(antsibot)
+to autogenerate a meeting summary that is then posted in the forum
+[Workflows and Logs](https://forum.ansible.com/c/workflow/15) category. We recommend that
+that you edit the topic to add your project tag and the `meeting` tag to improve findability.
+
+## Raise awareness about your project
+
+Use the following communication options to let people know about your Ansible project:
```

Comment:
How

Reply:
Not sure what you are asking here? Each item below shows ways they can communicate.